compilers/concrete-compiler/compiler/lib/Support/CompilationFeedback.cpp

```diff
@@ -6,60 +6,115 @@
 #include <cassert>
 #include <fstream>
 
-#include "llvm/Support/JSON.h"
-
 #include "concretelang/Support/CompilationFeedback.h"
 
+using concretelang::protocol::Message;
+
 namespace mlir {
 namespace concretelang {
 
-void CompilationFeedback::fillFromClientParameters(
-    ::concretelang::clientlib::ClientParameters params) {
+inline size_t bitWidthAsWord(size_t exactBitWidth) {
```

Comment:
The name sounds a bit like a type conversion routine for the word size of a specific architecture. Maybe `ceilPow2`?

Reply:
Actually this is dead code... I remove !